internal/shortener/shortener.go

```diff
@@ -59,6 +69,22 @@ func (us *URLShortener) GetHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (us *URLShortener) PingHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		http.Error(w, "Only GET requests are allowed!", http.StatusBadRequest)
+		return
+	}
+
+	ok := us.Storage.Ping()
+
+	if ok {
+		w.WriteHeader(http.StatusOK)
+		return
+	} else {
+		http.Error(w, "Connection error", http.StatusInternalServerError)
+	}
```

Comment:
Можно немного сократить эту запись. Вообще в общем случае стоит следовать принципу fail fast, когда негативный кейс обрабатывается как можно скорее. 
```suggestion
	if ok := us.Storage.Ping(); !ok {
	    http.Error(w, "Connection error", http.StatusInternalServerError)
	}
	w.WriteHeader(http.StatusOK)	
```